dockerfiles/clojure-1.12.0.Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM clojure:temurin-24-lein-alpine
+
+ENV CODECRAFTERS_DEPENDENCY_FILE_PATHS="project.clj"
+
+WORKDIR /app
+
+# .git & README.md are unique per-repository. We ignore them on first copy to prevent cache misses
+COPY project.clj /app
```

Comment:
@andy1li this is outdated - we no longer do the thing where we try to only copy files required for build (since often they'll depend on other files and we never know what all to include). Try using the `COPY /app --exclude .git README.md` technique (search other languages for exact syntax)